tests/c/enum.c

```diff
@@ -0,0 +1,28 @@
+//===--- enum.c - test input file for iwyu --------------------------------===//
+//
+//                     The LLVM Compiler Infrastructure
+//
+// This file is distributed under the University of Illinois Open Source
+// License. See LICENSE.TXT for details.
+//
+//===----------------------------------------------------------------------===//
+
+// Test that IWYU doesn't suggest to forward-declare C-style enumeration
+
+#include "enum-direct.h"
```

Comment:
I don't know, is there any reason to write `"tests/c/filename.h"` and then `// IWYU_ARGS: -I .` instead of this?

Reply:
There used to be a portability quirk whether the compiler looked up local paths relative to the current file, or relative to the current main source file (MSVC did one and GCC did the other, not sure now). At this point I think all mainstream compilers agree, but we add the include path mostly for consistency. There is a few particular testcases without it.

Also, I think `IWYU_ARGS` might be required in order for the file to be executed as a test. Or maybe not.

Reply:
It seems that modern versions of VS, GCC and clang search quoted files in the including file directory. Moreover, such a format supports navigation in VS without making a special project.
> ... we add the include path mostly for consistency

So, should I use `"tests/c/*.h"` format?